galaxy/importer/models.py

```diff
@@ -88,26 +88,31 @@ def __init__(self, branch, commit, format, contents,
 
 @attr.s(frozen=True)
 class BaseCollectionInfo(object):
-    """Represents collection_info metadata in collection manifest."""
+    '''Represents collection_info metadata in collection manifest.
+
+    Includes data validation expected when collection is built.
+    '''
 
     namespace = attr.ib(default=None)
     name = attr.ib(default=None)
     version = attr.ib(default=None)
-    license = attr.ib(default=None)
+    license = attr.ib(factory=list)
```

Comment:
(note for potential later discussion) I wouldn't mind changing 'license' attr to 'licenses' but that is something we can discuss later and in the collection spec. Reasons being a) it is a list b) the python builtin 'license' name conflict